mlir/include/mlir/Transforms/DialectConversion.h

```diff
@@ -151,15 +152,7 @@ class TypeConverter {
   ///       existing value are expected to be removed during conversion. If
   ///       `std::nullopt` is returned, the converter is allowed to try another
   ///       conversion function to perform the conversion.
-  ///   * std::optional<LogicalResult>(T, SmallVectorImpl<Type> &,
```

Comment:
Is this an unrelated change?

Reply:
I just noticed that this overload does not exist anymore.